### ISSUE TITLE
Can not access web routes protected with the auth Middleware 

### ISSUE THREAD

**Issue by byron-okta** [auth0/laravel-auth0#420]
### Checklist

- [X] This can be reproduced using [the quickstart sample application](https://github.com/auth0-samples/laravel).
- [X] I have looked at [the README](https://github.com/auth0/laravel-auth0/#readme) and have not found a solution.
- [X] I have looked at [the `docs` directory](https://github.com/auth0/laravel-auth0/blob/main/docs) and have not found a solution.
- [X] I have searched [previous issues](https://github.com/auth0/laravel-auth0/issues) and have not found a solution.
- [X] I have searched [the Auth0 Community](https://community.auth0.com/tag/laravel) and have not found a solution.
- [X] I agree to uphold [the Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Laravel Version

9

### SDK Version

Other (specify below)

### PHP Version

PHP 8.1

### Description

SDK Version: 7.9.1

I can not access a web route protected with the `auth` middleware. I'm getting an `ERR_TOO_MANY_REDIRECTS` error in the browser, and then I'm getting redirected to the home page instead of the protected web route. 

**Expected Behavior**

After successfully authenticating, I should be able to access a protected route.

### How can we reproduce this issue?

1. Clone the [GitHub repository](https://github.com/auth0-samples/laravel/tree/7.x/sample#getting-started) provided in the Auth0 Laravel Web App QS.

```bash
git clone https://github.com/auth0-samples/laravel auth0-laravel-quickstart
```

2. Install the project dependencies and configure the Auth0 Laravel SDK using the `auth0.app.json` or the `.env` files.

3. Run the application.

```bash
php artisan serve
```

4. Access the [/private](http:localhost:8000/private) web route that uses the `auth` middleware to require a user to be authenticated to access the route.

5. You'll be redirected to the Auth0 Universal Login page to log in. Enter the credentials of a valid user to log in.

6. Open the Developer tools and observe the network requests. You'll notice more than one redirect between the protected route `/private` and the login route `/login`:

![image](https://github.com/auth0/laravel-auth0/assets/116190812/e7af7add-0729-4831-9287-eb1d3db1f1ad)

**Result**

After successfully logging in, you'll see an `ERR_TOO_MANY_REDIRECTS` error in the browser, and then you'll be redirected to the home page instead of the protected `/private` web route. Even if you're already authenticated and try to access the protected `/private` route, you'll experience this same behavior.

**Comment by AndreiDecuseara** [comment]
Hi!
As a quick solution that I discovered:
In config file: config/auth.php change to this:
```
    'defaults' => [
        'guard' => 'auth0-session', //web
        'passwords' => 'users',
    ],
```
This version of auth0 seems to using `auth0-session` guard instead web.

**Comment by evansims** [comment]
Hi @byron-okta 👋 Thanks for the heads up, I've identified the issue and merged a fix (#423). It'll work as expected once I cut a new release. Want to check on a few more bugs before I ship it, but it'll be yet this week.

**Comment by byron-okta** [comment]
Thank you @evansims 💯
